### PR TITLE
Tested single server installation procedure on a fresh Ubuntu 20 server

### PR DESCRIPTION
**Pull request by inkuss**
Hi @marciafolio , I have just tested the installation procedure for the single server with containers and added my changes to this Pull Request. 
Other persons of SysOps has also tested it and found similar things (installation of perl modules is missing, wrong filename to increase "max connections" - should be /etc/postgresql/10/main/postgresql.conf, unclear in which directory you are in in some points of this prescription, Okapi version was not up-to-date [Okapi couldn't pull modules]). I corrected all these mistakes/weaknesses in this PR.

Please consider merging this pull request and/or have it reviewed by one of the tech writers.

### CODE REVIEW

File: content/en/docs/Getting started/Installation/singleserverwithcontainers.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ sudo apt-get -y install postgresql-10 postgresql-client-10 postgresql-contrib-10
 4. Configure PostgreSQL to listen on all interfaces and allow connections from all addresses (to allow Docker connections).
 
 * Edit the file **/etc/postgresql/10/main/postgresql.conf** to add line **listen_addresses = '*'** in the "Connection Settings" section.
-* Edit the file **/etc/postgresql/10/main/pg_hba.conf** to increase **max_connections** (e.g. to 500)
+* Edit the file **/etc/postgresql/10/main/postgresql.conf** to increase **max_connections** (e.g. to 500)
```
Comment (julianladisch) [inline content/en/docs/Getting started/Installation/singleserverwithcontainers.md:68]: This duplicates https://github.com/folio-org/docs/pull/8

File: content/en/docs/Getting started/Installation/singleserverwithcontainers.md [hunk 2]
```diff
@@ -29,44 +29,53 @@ A FOLIO instance is divided into two main components.  The first component is Ok
...
+
 ## Installing Okapi
 
 ### Okapi requirements
 
 1. Update the APT cache.
 
 ```
-sudo apt-get update
+sudo apt update
```
Comment (julianladisch) [inline content/en/docs/Getting started/Installation/singleserverwithcontainers.md:48]: Why is this change needed? `apt-get` works for me on Ubuntu 20.04:
```
docker run --rm -it ubuntu:20.04 sh
# apt-get update
Get:1 http://archive.ubuntu.com/ubuntu focal InRelease [265 kB]
...
```

Comment (inkuss) [reply content/en/docs/Getting started/Installation/singleserverwithcontainers.md:48]: apt-get works on Ubuntu 20. However, apt seems to be the more modern command. 
"apt consists some of the most widely used features from apt-get, apt-cache and apt-config leaving aside obscure and seldom used features."  (Source: https://askubuntu.com/questions/445384/what-is-the-difference-between-apt-and-apt-get)
See also https://itrig.de/index.php?/archives/2316-apt-vs.-apt-get-Was-ist-der-Unterschied.html .
It is not necessary to make this change here now, but mabye future manuals for Ubuntu will use apt rather than apt-get; we could also wait until this has turned out to be true.

File: content/en/docs/Getting started/Installation/singleserverwithcontainers.md [hunk 3]
```diff
@@ -98,21 +107,23 @@ Follow the instructions from official documentation for [docker](https://docs.do
 
 ```
 sudo curl -L \
-  "https://github.com/docker/compose/releases/download/1.26.2/docker-compose-$(uname -s)-$(uname -m)" \
+  "https://github.com/docker/compose/releases/download/1.27.4/docker-compose-$(uname -s)-$(uname -m)" \
```
Comment (julianladisch) [inline content/en/docs/Getting started/Installation/singleserverwithcontainers.md:110]: Have you encountered problems with the latest stable version https://github.com/docker/compose/releases/tag/1.28.6 ?

File: content/en/docs/Getting started/Installation/singleserverwithcontainers.md [hunk 4]
```diff
@@ -98,21 +107,23 @@ Follow the instructions from official documentation for [docker](https://docs.do
...
   -o /usr/local/bin/docker-compose
 sudo chmod +x /usr/local/bin/docker-compose
 ```
 
 8. Install Apache Kafka and Apache ZooKeeper.  Apache Kafka and Apache ZooKeeper are required by FOLIO [mod-pubsub](https://github.com/folio-org/mod-pubsub).  Both Kafka and ZoopKeepr are installed below using docker-compose.
 
+Take into account that you have to change the **KAFKA_ADVERTISED_LISTENERS** value for the private IP of your server, instead of 10.0.2.15 for a Vagrant box.
+
 ```
+vim ~/folio-install/runbooks/single-server/scripts/docker-compose-kafka-zk.yml  # chnage IP in KAFKA_ADVERTISED_LISTENERS
```
Comment (julianladisch) [inline content/en/docs/Getting started/Installation/singleserverwithcontainers.md:120]:
```suggestion
vim ~/folio-install/runbooks/single-server/scripts/docker-compose-kafka-zk.yml  # chnage IP in KAFKA_ADVERTISED_LISTENERS
```
```suggestion
vim ~/folio-install/runbooks/single-server/scripts/docker-compose-kafka-zk.yml  # change IP in KAFKA_ADVERTISED_LISTENERS
```

File: content/en/docs/Getting started/Installation/singleserverwithcontainers.md [hunk 5]
```diff
@@ -188,35 +208,17 @@ The content of registry.json should look like this:
 ```
 {
   "urls": [
-    "http://folio-registry.aws.indexdata.com"
+    "http://folio-registry.dev.folio.org"
```
Comment (julianladisch) [inline content/en/docs/Getting started/Installation/singleserverwithcontainers.md:211]:
```suggestion
    "https://folio-registry.dev.folio.org"
```
See https://github.com/folio-org/folio-install/blob/master/runbooks/single-server/scripts/okapi-pull.json

File: content/en/docs/Getting started/Installation/singleserverwithcontainers.md (hunk 3 above)
Comment (inkuss) [reply content/en/docs/Getting started/Installation/singleserverwithcontainers.md:110]: No. I don't remember where  I took the version nr from. I am just trying to be in sync with the Q3-2020 installation prescriptionb on https://github.com/folio-org/folio-install.

File: content/en/docs/Getting started/Installation/singleserverwithcontainers.md (hunk 5 above)
Comment (inkuss) [reply content/en/docs/Getting started/Installation/singleserverwithcontainers.md:211]: +1

**Review by marciafolio — COMMENTED** [review]
Accepting and merging changes.
